packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -115,25 +140,44 @@ function tallyResults(vwComponents, contentBuildWC) {
 /**
  * Match component-library React component uses in individual files
  *
+ * It's possible a file imports more than one component at the same time
+ * using syntax like:
+ * import { VaTextInput, VaModal } from ....
+ * or:
+ * import {
+ *  VaTextInput,
+ *  VaModal,
+ * } from ...
+ *
+ * When this is the case, match.matches[1] (componentName = m[1] below),
+ * will look something like 'VaTextInput,\n VaModal,\n'. With this,
+ * componentName needs to become an array of component names, and each of those need
+ * to be iterated over using componentRegex to retrieve an accurate count.
  * @return {Object[]} - The result of multiple searches
  *
  */
 function findUsedReactComponents(vwModules, regexPattern) {
   // Get a list of files which import components from the component library
   const componentLibraryImports = search(vwModules, regexPattern);
-
   return componentLibraryImports.reduce((allFilesCompUses, match) => {
     // For each import in each file, find the component that's being imported
     // and search that file for how many times that component is _used_.
     const fileComponentUses = match.matches.reduce((singleFileUses, m) => {
-      // First item will be the import match, second will be the use
-      const componentName = m[1];
-      const componentRegex = new RegExp(`<(${componentName})`, 'g');
-      // Search each file that imports a component for each use of that component
-      // and add the search results to the return array
-      return singleFileUses.concat(
-        search(readFile(match.path), componentRegex)
-      );
+      // First item will be the import match, second will be a string of component names
+      const componentNames = m[1];
```

Comment:
Originally `m[1]` was assuming a single string. With the new regex, even if it is just a single string like `VaTextInput`, we're transforming it to an array to handle the cases where there are more than one import. See the jsdoc comment above the function for a broader explanation.